lib/node_modules/@stdlib/string/base/replace-before/package.json

```
@@ -0,0 +1,70 @@
+{
+    "name": "@stdlib/string/base/replace-before",
+    "version": "0.0.96",
```

```suggestion
    "version": "0.0.0",
```

Individual packages are versioned independently of main project.